docs/configuration/commons.md

```diff
@@ -261,6 +264,37 @@ The configured status code ranges are inclusive; that is, in the above example,
 # interval = "30s"
 ```
 
+## Life Cycle
+
+Controls the behavior of Traefik during the shutdown phase.
+
+```toml
+[lifeCycle]
+
+# Duration to keep accepting requests prior to initiating the graceful
+# termination period (as defined by the `graceTimeOut` option). This
+# option is meant to give downstream load-balancer sufficient time to
```

Comment:
It should be either: "a downstream load-balancer" or "downstream load-balancers".

Reply:
Pluralized for the better.